PowerValidatedSolutions.psm1

```diff
@@ -9431,27 +9448,27 @@ Function Export-vROPsJsonSpec {
                                                         'properties'	= ($dataProperties | Select-Object -Skip 0)
                                                     }
                                                 }
-                                                $nodesobject += [pscustomobject]@{
-                                                    'type'			= "remotecollector"
-                                                    'properties'	= ($remoteCollector1Properties | Select-Object -Skip 0)
-                                                }
-                                                $nodesobject += [pscustomobject]@{
-                                                    'type'			= "remotecollector"
-                                                    'properties'	= ($remoteCollector2Properties | Select-Object -Skip 0)
+                                                if ($vcfVersion -ge "4.5.1") {
```

Comment:
Would this be simpler as the following - similar to the `deployOption` changes above?

```powershell
                                                if ($vcfVersion -ge "4.5.1") {
                                                    $type = "cloudproxy"
                                                } else {
                                                    $type = "remotecollector"
                                                }
                                                $nodesobject += [pscustomobject]@{
                                                    'type'			= $type
                                                    'properties'	= ($remoteCollector1Properties | Select-Object -Skip 0)
                                                }
                                                $nodesobject += [pscustomobject]@{
                                                    'type'			= $type
                                                    'properties'	= ($remoteCollector2Properties | Select-Object -Skip 0)
                                                }
```